.travis.yml

```diff
@@ -70,6 +70,7 @@ env:
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=1.7.4
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=2.0.2
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=2.1.1
+    - TRAVIS_FLAVOR=snmp FLAVOR_VERSION=latest
```

Comment:
Since this doesn't have a true version, I think we should remove the flavor_version. I'm not sure though, what do you think?